src/Persistence/AbstractQuery.php

```diff
@@ -53,11 +57,19 @@ public function __construct(Model $model)
         $this->limit = $model->limit;
     }
 
+    /**
+     * Find and return data from record with $id or NULL if none found.
```

Comment:
this comment is on edge, but programmers will imply this from the method signature - drop as well